src/interfaces/breadcrumb-interface.ts

```diff
@@ -2,4 +2,5 @@ export interface BreadcrumbIF {
   text: string
   to?: any
   href?: string
+  disabled?: boolean
```

Comment:
Whoops, this is a property I previously asked you to remove.

What's it for?

Reply:
Will disable the crumb manually, by passing the value into our vuetify comp prop.
 Since we don't want to control whether a local route or href is used, this allows manually disabling of links.
 Just gives us flexibility. Otherwise end user has no access to the base components prop.